e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @notifications
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Notifications', () => {
+    let user1;
+    let user2;
+    let team2;
+    let testTeam1TownSquareUrl;
+    let testTeam2TownSquareUrl;
+
+    const timestamp = Date.now();
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            testTeam1TownSquareUrl = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+
+            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({team: anotherTeam}) => {
+                team2 = anotherTeam;
+                testTeam2TownSquareUrl = `/${team2.name}/channels/town-square`;
+                cy.apiAddUserToTeam(team2.id, user1.id);
+                cy.apiAddUserToTeam(team2.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T560_1 Browser tab and team sidebar unreads and mentions - Mention in different team', () => {
+        let originalFavicon = '';
+
+        // # User 1 view team A
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+        cy.wait(TIMEOUTS.FIVE_SEC);
+
+        // # Remove all unreads before test
+        cy.get('#publicChannelList').should('be.visible').get('li').eq(1).click();
+
+        // * Check for no unreads or mentions
+        cy.get('.unread-title').should('not.exist');
+
+        // # Save path of current favicon path
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            originalFavicon = $path;
+        });
```

Comment:
Like I mentioned in the corresponding GH issue, better way is to compare favicon to base image. If you save this favicon (![favicon-default-16x16](https://user-images.githubusercontent.com/5334504/97292590-63900800-1886-11eb-9490-6a89f8938cbd.png)) as `/cypress/fixtures/favicon-default-16x16`, then you may compare the image data, like
```javascript
        cy.get('link[rel=icon]').should('have.attr', 'href').then((defaultFaviconUrl) => {
            cy.fixture('favicon-default-16x16.png').then((imageData) => {
                cy.request({url: defaultFaviconUrl, encoding: 'base64'}).then((response) => {
                    expect(response.status).to.equal(200);
                    expect(response.body).to.eq(imageData);
                });
            });
        });
```

Reply:
![image](https://user-images.githubusercontent.com/56567660/97397854-8b867680-192d-11eb-8685-351ce4ca1ccc.png)
Two image seems same (fixture(copied at images/favicon/favicon-default-16x16.png) and requested one), but has different base64 value.

Reply:
One way to verify is to open the `defaultFaviconUrl` in the browser. Let me know what you found. Or better yet, please share the code you have so far (can be in a gist).

Reply:
```
        // * Favicon should stay blue (not turn red) when there are mentions indicated
        cy.get('link[rel=icon]').should('have.attr', 'href').then((defaultFaviconUrl) => {
            cy.fixture('favicon-default-16x16.png').then((imageData) => {
                cy.request({url: defaultFaviconUrl, encoding: 'base64'}).then((response) => {
                    expect(response.status).to.equal(200);
                    expect(response.body).to.eq(imageData);
                });
            });
        });

        // * User 1 changes view to team 2
        cy.get(`#${team2.name}TeamButton`).click();
        cy.wait(TIMEOUTS.HALF_SEC);

        // * Favicon should turn red
        cy.get('link[rel=icon]').should('have.attr', 'href').then((defaultFaviconUrl) => {
            cy.fixture('favicon-mentions-16x16.png').then((imageData) => {
                cy.request({url: defaultFaviconUrl, encoding: 'base64'}).then((response) => {
                    expect(response.status).to.equal(200);
                    expect(response.body).to.eq(imageData);
                });
            });
        });
```
it fails on first time, and then success in restarted run.
Anyway, favicon is not turned red until I clicked team button which a mention posted.

Reply:
```
        // * User 1 changes view to team 2
        cy.get(`#${team2.name}TeamButton`).click();
        cy.wait(TIMEOUTS.HALF_SEC);

        // * Favicon should turn red
        cy.get('link[rel=icon]').should('have.attr', 'href').then((defaultFaviconUrl) => {
            cy.fixture('favicon-mentions-16x16.png').then((imageData) => {
                cy.request({url: defaultFaviconUrl, encoding: 'base64'}).then((response) => {
                    expect(response.status).to.equal(200);
                    expect(response.body).to.eq(imageData);
                });
            });
        });
```
favicon is not changed to red until test complete, and I browse into test session and click Team 2 button.
anyway, it was my mistake and I hide first comment.

Reply:
![20-10-28 04 20 26](https://user-images.githubusercontent.com/56567660/97404480-86c7bf80-1939-11eb-960c-4505dd880b5a.gif)
but favicon turns red when I simulated, it seems strange.